.github/workflows/branch-build.yml

```diff
@@ -99,10 +107,15 @@ jobs:
             --set e2eTest.image.repository="${TEST_IMAGE_REPOSITORY}" \
             --set e2eTest.image.tag="${IMAGE_TAG}" \
             --set e2eTest.slack.testerAppToken="${SLACK_TESTER_APP_TOKEN}" \
-            --set e2eTest.slack.additionalContextMessage="Branch test - commit SHA: ${GITHUB_SHA} - https://github.com/kubeshop/botkube/commit/${GITHUB_SHA}" \
+            --set e2eTest.slack.additionalContextMessage="Pull request: ${PR_NUMBER} - https://github.com/kubeshop/botkube/pull/${PR_NUMBER}" \
             --set e2eTest.discord.testerAppToken="${DISCORD_TESTER_APP_TOKEN}" \
             --set e2eTest.discord.guildID="${DISCORD_GUILD_ID}" \
-            --set e2eTest.discord.additionalContextMessage="Branch test - commit SHA: ${GITHUB_SHA} - https://github.com/kubeshop/botkube/commit/${GITHUB_SHA}"
+            --set e2eTest.discord.additionalContextMessage="Pull request: ${PR_NUMBER} - https://github.com/kubeshop/botkube/pull/${PR_NUMBER}"
 
-      - name: Run tests
-        run: "helm test botkube --namespace botkube --timeout=$INTEGRATION_TESTS_TIMEOUT --logs"
+      - name: Run ${{ matrix.integration }} tests
+        run: |
+          KUBECONFIG=$(k3d kubeconfig write ${{ matrix.integration }}-test-cluster) \
```

Comment:
If that's in the matrix strategy, I don't think that's necessary, but this is something to test on fork 👍

Reply:
The implementation has changed slightly. This is required for the `make` based integration tests to pick up the target K8s config.